<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>actionTmplDialog</class>
 <widget class="QDialog" name="actionTmplDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>420</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>420</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Zarządzanie szablonami</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="m_ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ActionDefinitionWidget" name="m_ActionWidget" native="true">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="m_Templates">
     <property name="minimumSize">
      <size>
       <width>280</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Szablony</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="1">
       <widget class="QPushButton" name="m_AddTemplate">
        <property name="text">
         <string>Dodaj</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="m_RemoveTemplate">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Usuń</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QListWidget" name="m_TemplateList">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="showDropIndicator" stdset="0">
         <bool>false</bool>
        </property>
        <property name="uniformItemSizes">
         <bool>true</bool>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="m_TemplateName">
        <property name="placeholderText">
         <string>Nazwa szablonu</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ActionDefinitionWidget</class>
   <extends>QWidget</extends>
   <header location="global">actionDefinitionWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_TemplateName</tabstop>
  <tabstop>m_AddTemplate</tabstop>
  <tabstop>m_RemoveTemplate</tabstop>
  <tabstop>m_TemplateList</tabstop>
  <tabstop>m_ButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_ButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>actionTmplDialog</receiver>
   <slot>apply()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>actionTmplDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>369</x>
     <y>339</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_AddTemplate</sender>
   <signal>clicked()</signal>
   <receiver>actionTmplDialog</receiver>
   <slot>addTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>194</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_RemoveTemplate</sender>
   <signal>clicked()</signal>
   <receiver>actionTmplDialog</receiver>
   <slot>removeTemplate()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_TemplateList</sender>
   <signal>currentItemChanged(QListWidgetItem*,QListWidgetItem*)</signal>
   <receiver>actionTmplDialog</receiver>
   <slot>currentTemplateChanged(QListWidgetItem*,QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addTemplate()</slot>
  <slot>removeTemplate()</slot>
  <slot>currentTemplateChanged(QListWidgetItem*,QListWidgetItem*)</slot>
  <slot>apply()</slot>
 </slots>
</ui>
